features/admin.sms-providers.v1/pages/twilio-sms-provider.tsx

```diff
@@ -185,7 +189,8 @@ const TwilioSMSProvider: FunctionComponent<TwilioSMSProviderPageInterface> = (
 };
 
 TwilioSMSProvider.defaultProps = {
-    "data-componentid": "twilio-sms-provider"
+    "data-componentid": "twilio-sms-provider",
+    isLoading: false
```

Comment:
defaultProps is going to be removed in a future react version, so let's not add new defaultProps.

Reply:
Fixed with 69799c37444d286ed33ae7f5f7db0f4e8be71823